.github/workflows/main.yml

```diff
@@ -0,0 +1,110 @@
+name: Main workflow
+
+# The event triggers are configured as following:
+# - on branch master, trigger the workflow on every push
+# - on any pull request, trigger the workflow
+# This is to avoid running the workflow twice on pull requests.
+on:
+  push:
+    branches:
+    - master
+  pull_request:
```

Comment:
Nice. This is the same at docs-kit, right?

Reply:
Yep